docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,89 @@
+
+## Introduction
+
+```{note}
+This page is for people developing Kedro, users working on their own Kedro projects should see the documentation for [deploying Kedro projects on Databricks](../deployment/databricks.md).
+```
+
+This guide describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
+
+## Prerequisites
+
+You will need the following to follow this guide:
+
+* Python **version >=3.8**.
+* An activated Python virtual environment into which you have installed the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html) with [authentication for your workspace](https://docs.databricks.com/dev-tools/cli/index.html#set-up-the-cli).
+* Access to a Databricks workspace with an [existing cluster](https://docs.databricks.com/clusters/create-cluster.html).
+* [GNU `make`](https://www.gnu.org/software/make/).
+* [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+* A local clone of the [Kedro git repository](https://github.com/kedro-org/kedro).
+
+## How to install a build of Kedro onto Databricks
+
+The development workflow for Kedro on Databricks is similar to the one for Kedro in general, when you develop and test your changes locally. The main difference comes when manually testing your changes on Databricks, since you will need to build and deploy the wheel file to Databricks to test it on a cluster.
+
+To make developing Kedro for Databricks easier, Kedro comes with a `Makefile` target named `databricks-build` that automates the process of building a wheel file and installing this on your Databricks cluster to save development time.
+
+### Setup the Databricks CLI to test a Kedro build
+
+Before you use `make databricks-build`, you must [set up the Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html#set-up-the-cli).
+
+The only remaining step is to create and environment variable with the ID of the cluster you are using to test your Kedro build. You can find the ID by executing the Databricks CLI command `databricks clusters list` and looking for the Cluster ID to the left of the name of your chosen cluster, for instance:
+
+```bash
+$ databricks clusters list
+1234-567890-abcd1234  General Cluster             TERMINATED
+0987-654321-9876xywz  Kedro Test Cluster          TERMINATED
+```
+
+In this case, the cluster ID of `Kedro Test Cluster` is `0987-654321-9876xywz`.
+
+Once you have determined the cluster ID, you must export it to an environment variable named `DATABRICKS_CLUSTER_ID`:
+
+```bash
+# Linux or macOS
+export DATABRICKS_CLUSTER_ID=<your-cluster-id>
```

Comment:
Would this be persistent between different shells?

Reply:
Setting an environment variable in this way makes it available only in the shell process in which it is defined. If a user wants to make the environment variable persistent it's quite easy to google a solution for their particular shell.